app/Http/Controllers/UnIgnoreProjectController.php

```diff
@@ -10,6 +10,12 @@ public function __invoke(Project $project)
     {
         $project->update(['ignored' => false]);
 
+				if (request()->wantsJson()) {
```

Comment:
I believe we will always be returning json so we can remove this conditional.

Reply:
Fixed with latest commit